### ISSUE TITLE
Server response handling in dialog

### ISSUE THREAD

**Issue by pejaham** [nakupanda/bootstrap3-dialog#190]
Similar question as the previous poster: when I post a form from within a page loaded in the dialog, I want the server response page in the same dialog (error/success messages) - not in a new page, something like iframe behavior. Is there a trick to make this possible?

**Comment by nakupanda** [comment]
Yes, it's surely possible, but you will need to implement things in ajax style rather than typical form submitting.
